external-crates/move/crates/move-compiler/src/unit_test/filter_test_members.rs

```diff
@@ -62,22 +65,22 @@ impl FilterContext for Context<'_> {
     }
 
     // A module member should be removed if:
-    // * It is annotated as a test function (test_only, test, random_test, abort) and test mode is not set; or
+    // * It is annotated as a test function (test_only, test, random_test, abort) and test mode is
+    //   not set; or
     // * If it is a library and is annotated as #[test]
     fn should_remove_by_attributes(&mut self, attrs: &[P::Attributes]) -> bool {
-        use known_attributes::TestingAttribute;
-        let flattened_attrs: Vec<_> = attrs.iter().flat_map(test_attributes).collect();
+        let flattened_attrs: Vec<_> = attrs.iter().flat_map(test_attribute_kinds).collect();
         let is_test_only = flattened_attrs.iter().any(|attr| {
             matches!(
                 attr.1,
-                TestingAttribute::Test | TestingAttribute::TestOnly | TestingAttribute::RandTest
+                AttributeKind_::Test | AttributeKind_::TestOnly | AttributeKind_::RandTest
```

Comment:
Should we add a function for this? Is there one? `is_test_only` or something similar?
Maybe `enabled_in_test_mode` or something more forward looking